typesense/src/test/resources/application.conf

```diff
@@ -0,0 +1,5 @@
+akka {
```

Comment:
not sure if this should be reference.conf as it's test scope but it's worth checking

Reply:
To be honest I don't know, I see in most modules it's called reference in main and application in test

Reply:
it makes sense to some degree, reference in main as all reference.conf files are merged and compacted to provide defaults and application.conf is the configuration that overrides defaults afaik

Reply:
`application.conf` is correct. This is the use-side of the configuration.